ITE/9/README.adoc

```diff
@@ -45,7 +45,7 @@ The link:https://github.com/in-toto/attestation/[in-toto Attestation Framework]
 
 When discussing a new attestation type, what is really being defined is a new type of link:https://github.com/in-toto/attestation/tree/main/spec#predicate[predicate]. The developer(s) of this new predicate must create a formal document that specifies the predicate type, and discusses several other related aspects. The purpose and prerequisites for the new attestation type MUST be defined. The document should also include a model that describes how the various components of the predicate map to supply chain semantics, juxtaposed with detailed use cases. These use cases SHOULD include example metadata that clearly highlight how the proposed predicate addresses the shortcoming described by them. Including these aspects will aid readers and future adopters understand the why and how of a particular attestation type.
 
-The specification MUST contain a detailed schema of the fields that make up the predicate. This schema MUST define unambiguously a unique `predicateType` following the rules defined by the attestation framework. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/main/spec/v1.0-draft#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
+The specification MUST contain a detailed schema of the fields that make up the predicate. This schema MUST define unambiguously a unique `predicateType` and a shorter `predicateName` following the rules defined by the attestation framework. The `predicateName` is expected to be used as the predicate specification's file name as well as a hint in certain contexts such as a media type to indicate the predicate embedded in the attestation. The name must not include version information and is expected to only be used as an unauthenticated hint. The schema must also clearly define the types of every field, with ranges for values where applicable. Further, the specification MUST contain rules for parsing the fields, as well as make recommendations for handling edge cases, where parsing or handling of fields would deviate from the standard link:https://github.com/in-toto/attestation/tree/v1.0/spec/v1.0#parsing-rules[parsing rules]. For example, it may be preferable in some cases to silently allow extra, undefined fields, while in other cases such metadata should be rejected.
```

Comment:
Why is the `predicateName` considered unathenticated? Is it not covered by a signature?

Reply:
This is not listed in the predicate itself. This is a change to the _specification_ of a predicate, like this document: https://github.com/in-toto/attestation/blob/main/spec/predicates/test-result.md. The proposal is to add, alongside the type URI, a field that basically says "Predicate name: `test-result`". This short name can then be used in media types and the like to provide an unauthenticated hint at the predicate type embedded in the corresponding attestation without getting into the version and other aspects. See: https://github.com/in-toto/attestation/issues/271

Reply:
I don't understand: why is it considered _unauthenticated_ when it seems to be in the signed portion of an attestation?

Reply:
This is _not_ in the attestation, this is in the markdown file that describes a predicate.

Reply:
I see, thanks for the clarification. This is a bit confusing TBH. Are the other fields that are in the schema but not in the actual predicate? If not, why not?

Reply:
ITE-9 is a process ITE that describes how users can submit a new predicate to be vetted. As such, it requires each predicate definition to include some specific information such as its type URI (used in the attestation statement), other fields specific to the predicate, use cases, etc. This PR merely adds an additional field, the predicate name, to that document. None of the things listed in ITE-9 go directly in a particular predicate, but rather the markdown file for each predicate in https://github.com/in-toto/attestation/tree/main/spec/predicates.

Reply:
I see. An ITE does seem overkill for this, but as long as we clearly explain that a predictName is metadata for a predicate scheme (which is already a bit confusing because the predicate is also metadata but on a lower level), then we should be good. Saying it is "an unauthenticated hint" is more confusing than illuminating because that hint isn't actually in the predicate, and it wouldn't be unathenticated if it was part of the a predicate.

Reply:
I specified "unauthenticated hint" to set expectations on how the predicate name should be used. Let me try and tweak the phrasing.

Reply:
Moved things around to connect unauthenticated hint and the text about using it in media types. Does that help?

My thinking was that this was fine as ITE-9 only features at the time of introduction of a predicate. Once it's vetted, the attestation + verification specs handle how and what must be verified. WDYT?

Reply:
I'm really not trying to be difficult, but I _still_ don't understand how the predicateName is going to drastically change the media type of what's in the specification file (aren't they all just [Markdown](https://github.com/in-toto/attestation/tree/main/spec/predicates) files?). I don't think I'm the best editor for this, so I'm going to defer to @marcelamelara here 😂

Reply:
@trishankatdatadog I haven't reviewed this PR yet. But maybe some extra context may help. A while back, we were asked, on the attestation side, to provide guidance on predicate-level media types. This is helpful in settings where consumers of attestations may need a hint about the contents of a DSSE without unpacking the entire envelope. We (the attestation maintainers) decided, for a few reasons, that a predicate name would be the most succinct way of defining unique predicate media type patterns for such use cases. The practice of choosing unique predicate names is already in place, but currently only implicitly, so this PR seeks to make that practice explicit.

Reply:
I think I finally (sorta) get it. Left some comments in that PR to get more clarifications. Thanks!

Reply:
@trishankatdatadog can you check the updated text?